### ISSUE TITLE
🐛 Toast dismiss button white on mobile device + dark mode

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#13334]
### 🐛 Bug and Expected Behaviour

The button to dismiss a toast is showing up as white on a mobile device in dark mode.

### 🕵️ Details

**Operating system**: Android 15
**Browser**: Chrome
**Colour mode**: Dark

### 📋 Steps to Reproduce

1. Login to the site on andorid
2. Navigate to a form
3. Save form
4. Observe the toast dismiss button being in white

### 📸 Screenshot

![Image](https://github.com/user-attachments/assets/d0e9878b-f143-471a-ba8e-128ec53b9ccb)

### 🙋‍♀️ Proposed Solution

Maybe make it blackl? Contact a designer for a better solution

### ✅ Acceptance Criteria

- [ ] Dismiss button is not white (some darker colour)
- [ ] Meets min. contrast ratio requirements
